src/plugins/kernel_install/systemd_bootloader.sh

```diff
@@ -0,0 +1,52 @@
+SYSTEMD_BOOT_ENTRIES_DIR="/boot/loader/entries"
+
+function run_bootloader_update()
+{
+  local flag="$1"
+  local target="$2"
+  local name="$3"
+  local sudo_cmd
+  local loader_conf="${SYSTEMD_BOOT_ENTRIES_DIR}/kw_${name}_linux.conf"
+
+  #if [[ ! -e "/boot/vmlinuz-${name}" && -e "$loader_conf" ]]; then
+  #  echo removing boot entry
+  #  remove_systemd_boot_entry "$flag" "$name"
+  #fi
+
+  if [[ -e "/boot/vmlinuz-${name}" ]]; then
+    add_systemd_boot_entry "$flag" "$target" "$name"
+  fi
+}
+
+function add_systemd_boot_entry()
+{
+  local flag="$1"
+  local target="$2"
+  local name="$3"
+  local loader_conf="${SYSTEMD_BOOT_ENTRIES_DIR}/kw_${name}_linux.conf"
+  local kernel_parameters
+
+  [[ "$target" == 'local' ]] && sudo_cmd='sudo'
```

Comment:
I think you forgot to use `sudo_cmd` for local deploy. We don't need `sudo` for remote deploy.